src/gradient.rs

```diff
@@ -411,9 +411,11 @@ impl<const N: usize, CS: ColorSpace> ColorStopsSource for [OpaqueColor<CS>; N] {
 
 #[cfg(test)]
 mod tests {
+    extern crate alloc;
     use super::Gradient;
+    use alloc::vec;
     use color::{cache_key::CacheKey, palette, parse_color};
-    use std::collections::HashSet;
+    use hashbrown::HashSet;
```

Comment:
So to clarify, when you're developing tests "locally" `std` will still be in scope (but without its prelude)?
So you can for example use `dbg!` locally, although you'd need to import it as `std::dbg`.

With the tradeoff being that when it gets to CI, anything from std you used will cause a compile error.
I don't think there's a better state practically possible here (ideally, there'd be a `clippy::disallowed_paths` lint we could configure here).

Reply:
We'd probably need a `#[cfg(test)]extern crate std;` ... and that'd give us `std::collections::HashSet` without a dev dependency ... which is tempting.